packages/react/__tests__/suppress-safari-class-name-mismatch.js

```diff
@@ -0,0 +1,64 @@
+import {
+  isPossiblyBenignClassNameMismatch,
+  suppressSafariClassNameMismatch
+} from '../src/suppress-safari-class-name-mismatch'
```

Comment:
since the implementation of this module doesn't focus on Safari I would rename it to not mention any engine - just focus on the TCO in the name